src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/Aes.cs

```diff
@@ -12,34 +10,20 @@ public abstract class Aes : SymmetricAlgorithm
     {
         protected Aes()
         {
-            this.BlockSize = 128;
-            this.KeySize = 256;
-            this.Mode = CipherMode.CBC;
-        }
-
-        public override KeySizes[] LegalBlockSizes
-        {
-            get
-            {
-                return s_legalBlockSizes.CloneKeySizesArray();
-            }
-        }
+            LegalBlockSizesValue = s_legalBlockSizes;
+            LegalKeySizesValue = s_legalKeySizes;
 
-        public override KeySizes[] LegalKeySizes
-        {
-            get
-            {
-                return s_legalKeySizes.CloneKeySizesArray();
-            }
+            BlockSizeValue = 128;
+            // TODO(12368): FeedbackSizeValue = 8;, but the field is not yet available.
+            KeySizeValue = 256;
+            ModeValue = CipherMode.CBC;
         }
 
         public static Aes Create()
         {
             return new AesImplementation();
         }
 
-
-
         private static readonly KeySizes[] s_legalBlockSizes = { new KeySizes(128, 128, 0) };
```

Comment:
I don't really care for this. Since this static is passed into base as a ref, any derived class can change the array for all instances (because the field is protected) and other derived types... It should be passed in via clone() or just do the new each time.